.github/workflows/release.yaml

```diff
@@ -0,0 +1,292 @@
+name: Release build
+
+on:
+  push:
+    tags:
+      - 1*
+
+jobs:
+
+  get-tag:
+
+    if: github.repository_owner == 'Autoware-AI'
+
+    runs-on: ubuntu-latest
+    outputs:
+      tag_ref: ${{ steps.get_tag.outputs.tag_ref }}
+
+    steps:
+      - name: Get tag
+        id: get_tag
+        run: |
+          TAG_REF=${{ github.ref }}
+          TAG_REF=`echo $TAG_REF | cut -c11-`
+          echo ::set-output name=tag_ref::$TAG_REF
+
+      - name: Echo tag
+        run: echo ${{ steps.get_tag.outputs.tag_ref }}
+
+
+  dockerfile-lint:
+
+    if: github.repository_owner == 'Autoware-AI'
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+      fail-fast: false
+
+    container:
+      image: hadolint/hadolint:latest-debian
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Linting
+        run: |
+          hadolint generic/Dockerfile
+          hadolint generic/Dockerfile.legacy.catkin
+          hadolint generic/Dockerfile.legacy.colcon
+          hadolint generic/Dockerfile.base
+          hadolint generic/Dockerfile.cuda.kinetic
+          hadolint generic/Dockerfile.cuda.melodic
+          hadolint crossbuild/Dockerfile.kinetic-crossbuild
+          hadolint crossbuild/Dockerfile.kinetic-crossbuild-driveworks
+          hadolint crossbuild/Dockerfile.melodic-crossbuild
+          hadolint crossbuild/Dockerfile.melodic-crossbuild-driveworks
```

Comment:
Probably don't need linting on the release jobs either.

Reply:
Removed